operator/cmd/mesh/install.go

```diff
@@ -236,9 +240,99 @@ func InstallManifests(iop *v1alpha12.IstioOperator, force bool, dryRun bool, res
 		return iop, err
 	}
 
+	if dryRun {
+		currentObj, err := reconciler.GetObject(iop)
+		if err != nil {
+			return iop, err
+		}
+		obj, err := object.ParseYAMLToK8sObject([]byte(iopStr))
+		if err != nil {
+			return iop, err
+		}
+		expectedObj := obj.UnstructuredObject()
+		result, err := showDifferences(currentObj, expectedObj)
+		if err != nil {
+			return iop, err
+		}
+		fmt.Println(result)
+	}
 	return iop, saveIOPToCluster(reconciler, iopStr)
 }
 
+func showDifferences(existing, target *unstructured.Unstructured) (string, error) {
+	// We are only interested in seeing the differences between the specs
```

Comment:
Why? label, annotation, etc changes can be just as important

Reply:
@howardjohn This is after discussion with @ostromart, if other parts of differences are desirable as well, we can include these entities. I am totally open on this one.

Reply:
@howardjohn John, now we are doing full everything comparison. Please see the new commit. Thanks.